### PR TITLE
239 cleaned sfx

### PR DESCRIPTION
**Pull request by CrispyKorn**
Added sound effects that play when an area is cleaned. Works with the new TaskTracker to play the sounds when an entire scene is cleaned, and also plays victory music when the whole level is complete.

THERE's NO TIME FOR A CHECKLIST JUST TRUST ME BRO IT WORKS

### CODE REVIEW

File: MonstieWash/Assets/Scripts/Erasing/Tasks/TaskTracker.cs [hunk 1]
```diff
@@ -5,17 +5,25 @@
```
Comment (102101141) [inline MonstieWash/Assets/Scripts/Erasing/Tasks/TaskTracker.cs:5]:
```suggestion
[RequireComponent(typeof(RoomSaver))]
[RequireComponent(typeof(UIManager))]
[RequireComponent(typeof(SoundPlayer))]
```

Comment (102101141) [reply MonstieWash/Assets/Scripts/Erasing/Tasks/TaskTracker.cs:5]: As the number of components required by the class increases some error protection could be useful to stop those pesky designers

**Review by 102101141 — APPROVED** [review]
I feel like I'm levelling up in an MMO with these sounds, lol :)
Code looks great, no problems with the changes. Made a suggestion for designer error-proofing but other than that looks good to go! 
One note though, for the designers, is there a way to make the relationship between the Sound Player and the UI Manager scripts clearer? I'll attach a screenshot of what I mean, but if the Sparkle sound effect wasn't included on the SoundPlayer I would personally have no idea what it's used for. 
Screenshot 2024-05-24 234106

**Comment by CrispyKorn** [comment]
Yeah I getcha, you're right it is a bit messy but I didn't wanna break more stuff by moving things around too much, so I'll fix that all up once everything is in by everyone.

**Pull request merged**